### ISSUE TITLE
EnableMockImageService not working on Mock Tests (working in plain Unit Tests)

### ISSUE THREAD

**Issue by lhughey** [luberda-molinet/FFImageLoading#1244]
### Description

**I would be happy to submit a PR to help fix this issue**, but I need some help to identify the problem. The error code I'm receiving is **You are referencing the Portable version in your App - you need to reference the platform version**

**Please note that using the ImageService.EnableMockImageService does not work in this scenario**. The difference in the samples is that you are referencing a method within the ImageServer.Instance, vs creating a control with XAML (similar to what a Mock does). 

I've previously opened up an issue at 
https://github.com/luberda-molinet/FFImageLoading/issues/1143
there was also another ticket opened at https://github.com/luberda-molinet/FFImageLoading/issues/1174
Both tickets were closed, but this issue is not resolved by the ImageService.EnableMockImageService = true declaration.

The reason this important:
I am creating mock services for my login process and FFImageLoading is breaking the mock tests. I cannot create mocks while using FFImageLoading, and it one of my more useful components.

### Steps to Reproduce

1. Create a Xamarin Mock Unit Test project.
2. Drop this code in as a test

Use this test
```
[Fact]
public void DashboardPage_CreateApp()
{
	ImageService.EnableMockImageService = true;
	var svgXaml =
		"<SvgCachedImage xmlns:forms=\"clr-namespace:FFImageLoading.Svg.Forms;assembly=FFImageLoading.Svg.Forms\"  Source=\"resource://Project.Common.Images.Svgs.icon_3.svg?assembly=Project.Common\" HeightRequest=\"24\"/>";
   
	SvgCachedImage image = new SvgCachedImage();
	image.LoadFromXaml(svgXaml);
	Assert.Equal(24,image.HeightRequest);
}
```

### Expected Behavior

The test should pass

### Actual Behavior

The test throws an "You are referencing the Portable version in your App - you need to reference the platform version"

### Basic Information

- Version with issue: 2.4.3.840 
- Last known good version: unknown
- Platform: Unit testing on Visual Studio 2017, using Xamarin forms mock

**Comment by praveenkrjha** [comment]
I am sailing in the same boat.

**Comment by daniel-luberda** [comment]
Should be fixed now. Please try new prerelease and let me know if it helped.

**Comment by lhughey** [comment]
Will do. Thanks.

**Comment by praveenkrjha** [comment]
I updated to latest pre release package "2.4.5.900-pre". Got past the "You are referencing the Portable version in your App - you need to reference the platform version" error but now it throws:

> Please call CachedImageRenderer.Init method in a platform specific project to use FFImageLoading!

How do we call CachedImageRenderer.Init in a unit test project?

**Comment by praveenkrjha** [comment]
Thanks Daniel. This issue is fixed in "2.4.5.907-pre".
